ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -225,6 +225,35 @@
 | {{ evpn_hostflap_detection_state }} | {{ evpn_hostflap_detection_window }} |  {{ evpn_hostflap_detection_threshold }} |
 {%         endif %}
 {%     endif %}
+{%     set evpn_gw_config = namespace(peer_groups = [], configured = false) %}
+{%     for peer_group in router_bgp.peer_groups | arista.avd.natural_sort %}
+{%         if router_bgp.address_family_evpn.peer_groups[peer_group].domain_remote is arista.avd.defined(true) %}
+{%             do evpn_gw_config.peer_groups.append(peer_group) %}
+{%             set evpn_gw_config.configured = true %}
+{%         endif %}
+{%     endfor %}
+{%     if router_bgp.address_family_evpn.neighbor_default.next_hop_self_received_evpn_routes.enable is arista.avd.defined(true) %}
+{%         set evpn_gw_config.configured = true %}
+{%     endif %}
+{%     if router_bgp.address_family_evpn.neighbor_default.next_hop_self_received_evpn_routes.inter_domain is arista.avd.defined(true) %}
+{%         set evpn_gw_config.configured = true %}
+{%     endif %}
+{%     if evpn_gw_config.configured is arista.avd.defined(true) %}
+
+#### EVPN DCI Gateway Summary
+
+| Settings | Value |
+| -------- | ----- |
+{%         if evpn_gw_config.peer_groups | length > 0 %}
```

Comment:
You can remove this condition as you have already done a test before on line 241, ```if evpn_gw_config.configured is arista.avd.defined(true) ``` Both the tests are same.